docs/user.md

```diff
@@ -425,3 +425,35 @@ monitoring is outside the scope of operator responsibilities. See
 [configuration reference](reference/cluster_manifest.md) and
 [administrator documentation](administrator.md) for details on how backups are
 executed.
+
+## TLS configuration
+
+By default self-signed certificates are generated when the postgres container
+starts up. But it's also possible to provide your own secrets like so:
+
+Generate a new cert:
```

Comment:
I don't think you need to document how to create the cert, just how to add it to kubernetes with `kubectl create secret` (as you do below) and how to configure the cluster.

Reply:
ok, removed